### ISSUE TITLE
Do not log "invalid request token" exceptions

### ISSUE THREAD

**Issue by leofeyer** [contao/core-bundle#1139]
@contao/developers How can we stop certain exceptions such as "invalid request token" from cluttering up the log files?

**Comment by aschempp** [comment]
By not logging them? 😂 
https://github.com/contao/core-bundle/blob/master/src/EventListener/PrettyErrorScreenListener.php#L329

**Comment by leofeyer** [comment]
You mean a hardcoded list? What if someone wants to add more entries?

**Comment by aschempp** [comment]
We could make it configurable in the bundle. But how would you prevent logging in a regular Symfony application?

**Comment by leofeyer** [comment]
The more I think about it, I come to the conclusion that we should only log unknown exceptions.

Do not log the known exceptions:

```php
    private static $mapper = [
        ForwardPageNotFoundException::class => 'forward_page_not_found',
        IncompleteInstallationException::class => 'incomplete_installation',
        InsecureInstallationException::class => 'insecure_installation',
        InvalidRequestTokenException::class => 'invalid_request_token',
        NoActivePageFoundException::class => 'no_active_page_found',
        NoLayoutSpecifiedException::class => 'no_layout_specified',
        NoRootPageFoundException::class => 'no_root_page_found',
    ];
```

And do not log the back end exceptions, because both the known exceptions and the back end exceptions are primarily user error which provide little value when added in the log file, don't you think?

**Comment by leofeyer** [comment]
Well, we should log the `ForwardPageNotFoundException::class` exception, because this is a configuration error that needs to be fixed.

**Comment by Toflar** [comment]
I agree, only `ForwardPageNotFoundException` is really relevant here.

**Comment by leofeyer** [comment]
Implemented in f72251fbb0386a9ac813c538ec0ffd69322bf482.
